### DISCUSSION TITLE
rcs-na-iphone-z-ios-18-stracone-nadzieje/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#127] [General]
# rcs-na-iphone-z-ios-18-stracone-nadzieje/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/rcs-na-iphone-z-ios-18-stracone-nadzieje/

**Comment by spexpl** [comment]
Szkoda tylko, iż tak trudno korzystać z RCS na Androidzie :/
